README.md

```diff
@@ -108,11 +109,19 @@ some_ros_package
 |   | <b>my_node.cpp</b>
 </pre>
 
+## Launch files
```

Comment:
:grimacing: can't believe I didn't put a section on this in the readme..... good idea! :smiley: